frontend/sw360-portlet/src/main/webapp/html/components/editRelease.jsp

```diff
@@ -244,6 +246,18 @@
 
         $('#formSubmit').click(
             function() {
+                var valid = true;
+                $.each($('#externalIdsTable tr.bodyRow'), function (i, row) {
+                    var key = $('#'+row.id+' td input.keyClass').val();
+                    var value = $('#'+row.id+' td input.valueClass').val();
+                    var isOneEmpty = ((key !== '') && (value === '')) || ((key === '') && (value !== ''));
```

Comment:
I don't mean to nitpick, but I still have a question.
`isOneEmpty` will be `true` if one and only one from both variables is empty. What happens if both are empty?

Reply:
If both are empty nothing will be added to the database so there's no issue and the release can just update - @maxhbr suggested that this would make things easier if someone wants to create an external id and then just changes their mind.

Reply:
Ok, fine. I got it